build/windows/installer/conf/fluent-bit-telegraf-tcp.conf

```diff
@@ -0,0 +1,9 @@
+[INPUT]
```

Comment:
I think you forgot to remove this from the common conf file

Reply:
> I think you forgot to remove this from the common conf file

yep, just fixed.